src/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+set(TARGET piTest)
+set(SOURCES
+	piTest.c
+	piControlIf.c
+)
+
+add_executable(${TARGET} ${SOURCES})
+target_include_directories(${TARGET} PRIVATE ../inc ../lib/piControl)
+
+install(TARGETS ${TARGET} DESTINATION usr/bin)
+# https://stackoverflow.com/a/41037224
+# TODO: this disregards any prefix set for installing
+install(CODE "execute_process( \
+	COMMAND ${CMAKE_COMMAND} -E create_symlink \
+	usr/bin/piControlReset usr/bin/${TARGET})")
```

Comment:
This is not the correct symlink. The original symlink to relative so this symlink needs to be relative as well.
Also prefixing is mandatory. Don't install to usr.
Maybe it would be better to create the symlink in the build directory and install the symlink along the binary in the install step.

Reply:
> Maybe it would be better to create the symlink in the build directory

I'm unsure how this works. It seems that, if I'm understanding this correctly, because `DESTINATION` is not given, `${CMAKE_INSTALL_PREFIX}` does not honor the `--prefix` flag in `cmake --install . --prefix $PREFIX` when using the `install` directive in lines 13-15. Just using `${CMAKE_INSTALL_BINDIR}/piControlReset` yields `bin/piControlReset`, not `$PREFIX/bin/piControl` like it does for the install target above.

If I'm building the file into the build directory, how would I refer to that file to install it? `install(FILES ...)` refers to files in the source directory, and hard coding the build directory is not an option.

Reply:
The target of the symlink is allways `./piTest`. The symlink like the executeable are installed to the same directory.

So it should not a big issue to create a symlink in the build directory along with the executeable. And in the install step it should be possible to install the symlink like any other file (maybe).